README.md

```diff
@@ -44,6 +44,35 @@ Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduc
 
 Please report issues [here](https://github.com/fuse-open/fuse-studio/issues).
 
+## Building installers
+
+Note that building installers is due to legacy reasons a bit cumbersome, and we eventually want that part of the build process cleaned up.
+
+### Making a macOS installer
+
+Note that the current installer scripts expects a signing key available in the keychain, which is the property of Fusetools AS. It should be easy to modify script to use a different signing key if desirable.
+By setting env var `SIGN` to `0` it should be possible to just create an unsigned installer, but haven't tested that this works.
+
+To give the installer a specific version, set the environment vars RELEASE_VERSION and BUILD_NUMBER (this used to be set by the old CI setup when running on TC).
+
+```shell
+git clean -xdf && RELEASE_VERSION=1.9.0-rc3 BUILD_NUMBER=1.9.0-rc3 Installer/OSX/build.sh
+```
+
+The above commands produces an installer located at `Installer/OSX/Fuse_1_9_0-rc3.pkg`. We'll rename this to `fuse_osx_1_9_0_rc2.pkg` before uploading the release, to be consistent with naming of older versions.
+
+The reason we do a `git clean` before building installer, is to avoid accidentially include cruft from older builds.
+
+### Making a Windows installer
+
+Making the Windows installer for the 1.9.0-rc3 release was done using the following commands (in a git bash shell).
```

Comment:
(If i read the script correctly pack.sh also requires you to have 7-Zip installed in `C:\Program Files\7-Zip\7z.exe`, though that is probably obvious from the error you'd get without it)

Reply:
Nice catch, adding a notice about this. 👍